### PR TITLE
Try to taint server resources on patch

### PR DESCRIPTION
**Pull request by JohnGarbutt**

### CODE REVIEW

File: roles/cluster_infra/tasks/main.yml [hunk 1]
```diff
@@ -45,6 +45,35 @@
     - providers.tf
     - resources.tf
 
+- name: Find terraform outputs, just to initi tofu state
```
Comment (JohnGarbutt) [inline roles/cluster_infra/tasks/main.yml:48]: We probably should extract all of this into something we can use more easily for other appliances.

**Review by mkjpryor — CHANGES_REQUESTED** [review]
I thought we said we were going to put this in the Terraform collection so it could easily be used elsewhere?

**Comment by JohnGarbutt** [comment]
> I thought we said we were going to put this in the Terraform collection so it could easily be used elsewhere?

We are, but @assumptionsandg is going to look at doing that next :)

**Comment by mkjpryor** [comment]
Superseded by #31

**Pull request closed without merging**